### PR TITLE
[#399] removed fraudulent demand regex

### PR DESCRIPTION
**Pull request by Samuel-Campbell**
- [x] Remove demand regex (bonus task)
- [x] Change additional_indemnity_money to boolean
- [x] Removed additional indemnity money regressor
- [x] Test pipeline from beginning to end (make sure nothing breaks) - using sample of 1000
- [x] Removed outcomes with low occurrence 

1- Removing demand regex makes tagging quicker. They are not used anyways
2- Additional indemnity money will now become a boolean since we will be calculating it's value using a formula that we know of
3- Had to remove outcomes with low occurrence --> delcares housing inhabitable. These outcomes cannot be implemented without a custom multi class svm wrapper. Right now the default one from sklearn doesn't always work. Since the training and testing sets are randomly split AND the outcome occurrence is very low THEN there is a chance that there are 0 values for a given outcome in the training set. If this happens then no classifier is created. You will get null pointer errors from this in the future.

### CODE REVIEW

File: src/ml_service/data/test/regex/orders_landlord_notify_tenant_when_habitable.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-ORDONNE au locateur sous toute peine que de droit, d'aviser la locataire dès que le logement sera redevenu propre à l'habitation
```
Comment (TaimoorRana) [inline src/ml_service/data/test/regex/orders_landlord_notify_tenant_when_habitable.txt:1]: This should pass. Why are we removing this?

File: src/ml_service/feature_extraction/post_processing/regex/misc/regex_lib_helper_test.py [hunk 2]
```diff
@@ -11,4 +11,5 @@ def test_regex_finder(self):
         self.assertTrue('tenant_continuous_late_payment' in regex_lib_helper.regex_finder(sentence))
 
     def test_get_regexes(self):
-        self.assertIsNotNone(regex_lib_helper.get_regexes('demand_lease_modification'))
+        regex_lib_helper.get_regexes('additional_indemnity_money')
+        self.assertIsNone(None)
```
Comment (TaimoorRana) [inline src/ml_service/feature_extraction/post_processing/regex/misc/regex_lib_helper_test.py]: will never fail

File: src/ml_service/model_training/regression/multi_output/multi_output_regression.py [hunk 3]
```diff
@@ -78,7 +74,7 @@ def predict(self, facts, outcomes):
             if outcomes[i] == 1:
                 column_name = MultiOutputRegression.classifier_labels[i][0]
                 if column_name == 'additional_indemnity_money':
-                    outcomes[i] = AdditionalIndemnityMoney().predict(facts)[0][0]
+                    outcomes[i] = 1 # placeholder
```
Comment (arekmano) [inline src/ml_service/model_training/regression/multi_output/multi_output_regression.py:77]: ?

Comment (arekmano) [reply src/ml_service/model_training/regression/multi_output/multi_output_regression.py:77]: will the business logic be on the ML Service side? If not, we can remove this attribute completely

**Review by arekmano — APPROVED** [review]
Looks good. Minor change

File: src/ml_service/data/test/regex/orders_landlord_notify_tenant_when_habitable.txt (hunk 1 above)
Comment (Samuel-Campbell) [reply src/ml_service/data/test/regex/orders_landlord_notify_tenant_when_habitable.txt:1]: i removed the regex

File: src/ml_service/model_training/regression/multi_output/multi_output_regression.py (hunk 3 above)
Comment (Samuel-Campbell) [reply src/ml_service/model_training/regression/multi_output/multi_output_regression.py:77]: we need thijs to keep the app functional

File: src/ml_service/feature_extraction/post_processing/regex/regex_lib.py [hunk 4]
```diff
@@ -12,220 +12,6 @@ class RegexLib:
...
-        ], "MONEY_REGEX"),
-        ("tenant_eviction", [
-            re.compile(
-                r".+" + LANDLORD_REGEX +
-                r".*(expulsion|éviction).*" + TENANT_REGEX + r"",
-                re.IGNORECASE
-            )
-        ], "BOOLEAN")
-    ]
-
```
Comment (dodoels) [inline src/ml_service/feature_extraction/post_processing/regex/regex_lib.py:228]: I feel like it might be useful for CJLs research. Maybe comment it out?

**Review by dodoels — COMMENTED** [review]
I was gonna say system test before merging... but ok

**Pull request merged**